packages/cli/lib/lib/webpack/transform-config.js

```diff
@@ -130,35 +130,15 @@ module.exports = async function (env, webpackConfig, isServer = false) {
 	}
 };
 
-/**
- * WebpackConfigHelpers
- *
- * @class WebpackConfigHelpers
- */
```

Comment:
I removed this JSDoc comments as they weren't particularly descriptive of types and, as far as I know how to wrangle the TS language server, there's no way they'd be picked up on in any user project.

Reply:
Won't these help if you use typescript to check these JS files?

Reply:
Hm, well they could help ensure you're returning the correct type, but, at least to my knowledge, without a js/tsconfig set up the TS language server won't type check.

In my editor, I'd need to create a `jsconfig.json` in the project in order for say `getPluginsByType` to show an error when I replace the return value with `'foo'`.

This might be editor-specific though. I'll revert this, and just switch them over to use the types created in this PR (as they're more specific)

Reply:
Sounds good